gemdaq-testing/gemhardware/src/common/amc13/AMC13Manager.cc

```diff
@@ -109,28 +150,37 @@ void gem::hw::amc13::AMC13Manager::actionPerformed(xdata::Event& event)
   m_enableFakeData     = m_amc13Params.bag.enableFakeData.value_;
   m_monBackPressEnable = m_amc13Params.bag.monBackPressure.value_;
   m_enableLocalTTC     = m_amc13Params.bag.enableLocalTTC.value_;
-  m_enableLocalL1A     = m_amc13Params.bag.enableLocalL1A.value_;
-  m_internalPeriodicPeriod = m_amc13Params.bag.internalPeriodicPeriod.value_;
-  m_L1Amode            = m_amc13Params.bag.l1Amode.value_;
-  m_L1Arules           = m_amc13Params.bag.l1Arules.value_;
-  m_L1Aburst           = m_amc13Params.bag.l1Aburst.value_;
-  m_sendL1ATriburst    = m_amc13Params.bag.sendl1ATriburst.value_;
-  m_startL1ATricont    = m_amc13Params.bag.startl1ATricont.value_;
-
-  m_enableCalpulse     = m_amc13Params.bag.enableCalpulse.value_;
-  m_bgochannel         = m_amc13Params.bag.bgochannel.value_;
-  m_bgocmd             = m_amc13Params.bag.bgocmd.value_;
-  m_bgobx              = m_amc13Params.bag.bgobx.value_;
-  m_bgoprescale        = m_amc13Params.bag.bgoprescale.value_;
-  m_bgorepeat          = m_amc13Params.bag.bgorepeat.value_;
-  m_bgolong            = m_amc13Params.bag.bgolong.value_;
+
+  m_enableLocalL1A     = m_l1AConfig.at(0).bag.enableLocalL1A.value_;
+  m_internalPeriodicPeriod = m_l1AConfig.at(0).bag.internalPeriodicPeriod.value_;
+  m_L1Amode            = m_l1AConfig.at(0).bag.l1Amode.value_;
+  m_L1Arules           = m_l1AConfig.at(0).bag.l1Arules.value_;
+  m_L1Aburst           = m_l1AConfig.at(0).bag.l1Aburst.value_;
+  m_sendL1ATriburst    = m_l1AConfig.at(0).bag.sendl1ATriburst.value_;
+  m_startL1ATricont    = m_l1AConfig.at(0).bag.startl1ATricont.value_;
+
```

Comment:
These individual variables are probably no longer necessary, as the configuration should be coming from the bag (m_l1AConfig)